### ISSUE TITLE
Reproducing Results

### ISSUE THREAD

**Issue by galyona** [hardikbansal/Fader-Networks-Tensorflow#2]
Hello!

I've fixed a few small bugs in the code (perhaps version compatibility issues) but could not get the training to converge.
Were you able to reproduce the results in the original article using your code?
If so, could you post some guidelines?

Thanks!

**Comment by hardikbansal** [comment]
No, I haven't reproduced the results.

Have you read the code. Did you find any error or anything that I have not handled correctly?

Thanks,
Hardik

**Comment by LynnHo** [comment]
![image](https://user-images.githubusercontent.com/16457298/32038511-e2fa7ec6-ba5c-11e7-900c-446a9c75d8f4.png)
 
@hardikbansal I think it's not suitable to invert the attribute label for training the encoder. It should be more a sensible way, but I can't find it in the paper.

**Comment by hardikbansal** [comment]
Yup, in paper they have flipped the attributes rather have defined two attributes for the same thing. I haven't tried changing it yet.

**Comment by hardikbansal** [comment]
@LynnHo Did you find any other problem in the implementation that you think might be the cause this model is not working?

**Comment by LynnHo** [comment]
@hardikbansal Sorry, I haven't find out any solution yet.

**Comment by hardikbansal** [comment]
I have made necessary changes. There was an error in loss function and also the representation of attributes. I was not converting the attributes from -1,1 domain to 0,1 domain. I have not run it yet though, as i don't have enough resources to run it as of now.
